inductiva/fluids/simulators/openfoam.py

```diff
@@ -44,10 +44,9 @@ def run(
     def run_async(
         self,
         input_dir: types.Path,
-        method_name: str,
+        commands: List[dict],
```

Comment:
Since this is going to be accessible from a low level Openfoam calls, is there any security check done to the commands?

Reply:
Not yet, that will be done on the backend, as for the other simulators, this is still a security concern.